config/routes.rb

```diff
@@ -142,10 +142,11 @@
       get   "/settings" => "communities#settings",        as: :settings
       patch "/settings" => "communities#update_settings", as: :update_settings
 
+      get "getting_started_guide(/*all)" => "getting_started_guide#index", as: :getting_started_guide
```

Comment:
if I run "rake routes | grep getting_started_guide" I get
```
styleguide_getting_started_guide GET      /styleguide/getting_started_guide(/*all)(.:format)                                                                           admin_getting_started_guide GET (/:locale)/admin/getting_started_guide(/*all)(.:format)
````

However, if I rename those "as: :styleguide_getting_started_guide" and "as: :admin_getting_started_guide", I get:
```
styleguide_styleguide_getting_started_guide GET      /styleguide/getting_started_guide(/*all)(.:format)
admin_admin_getting_started_guide GET      (/:locale)/admin/getting_started_guide(/*all)(.:format)  
```

I'd like to avoid that double admin_admin_route_name thing